Products/CMFPlone/Portal.py

```diff
@@ -61,7 +61,7 @@ def __getattr__(self, name):
 
     def __setattr__(self, name, obj):
         # handle re setting an item as an attribute
-        if self._tree is not None and name in self:
+        if not name.startswith("_") and self._tree is not None and name in self:
```

Comment:
Shouldn't we be more strict with this change and check for `name.startswith("_p_") or `name.startswith("_v_")``?